.github/workflows/update.yml

```diff
@@ -21,19 +21,22 @@ jobs:
           yarn
         env:
           YARN_ENABLE_IMMUTABLE_INSTALLS: false
-
+      - name: Generate token
+        uses: tibdex/github-app-token@3beb63f4bd073e61482598c45c71c1019b59b73a #v2.1.0
+        id: generate_token
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          private_key: ${{ secrets.APP_PRIVATE_KEY }}
       - name: Create PR
         id: create-pr
-        uses: peter-evans/create-pull-request@v3
+        uses: peter-evans/create-pull-request@5e914681df9dc83aa4e4905692ca88beb2f9e91f # v7.0.5
         with:
-          token: ${{ secrets.CREATE_PR_ACCESS_TOKEN }}
-          commit-message: 'fix: update Carbon 11 compatible versions to latest'
-          committer: GitHub <noreply@github.com>
-          author:
-            ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>
-          branch: 'update-packages'
+          token: ${{ steps.generate_token.outputs.token }}
+          commit-message: 'build(deps): update Carbon 11 compatible versions to latest'
```

Comment:
THANK YOU for changing the convention _away_ from `fix` although I usually consider dependencies a chore?

Reply:
Haha yeah `fix` never felt correct. I'll update to `chore`

Reply:
`ci` might be even more appropriate https://github.com/pvdlg/conventional-commit-types?tab=readme-ov-file#commit-types